internal/supervisor/installation.go

```diff
@@ -112,7 +113,9 @@ type InstallationSupervisor struct {
 // cluster resources.
 type InstallationSupervisorCache struct {
 	initialized         bool
+	running             bool
```

Comment:
I think just closing the `stop` channel would be a bit simpler from a code perspective but I will leave it up to you.